packages/taquito-signer/README.md

```diff
@@ -0,0 +1,93 @@
+# Taquito Signer package
+
+**Storing private keys in memory is suitable for development workflows but risky for
+production use-cases! Use the InMemorySigner appropriately given your risk profile**
+
+`@taquito/signer` is an npm package that provides developers with signing functionality for Taquito. It can be injected as follows:
+
+### Loading an unencrypted private key
+
+If you configure Taquito this way, you will now be able to use every function that needs signing support.
+
+```js
+import { InMemorySigner } from '@taquito/signer';
+import { TezosToolkit } from '@taquito/taquito';
+
+const Tezos = new TezosToolkit('https://YOUR_PREFERRED_RPC_URL');
+
+Tezos.setProvider({ signer: await InMemorySigner.fromSecretKey('your_private_key') });
+```
+
+### Loading an encrypted private key with a passphrase
+
+If your private key is encrypted, you can specify a passphrase to decrypt it. Doing so will automatically decrypt the key and allow you to use the signer to sign transactions.
+
+```js
+import { InMemorySigner } from '@taquito/signer';
+import { TezosToolkit } from '@taquito/taquito';
+
+const Tezos = new TezosToolkit('https://YOUR_PREFERRED_RPC_URL');
+Tezos.setProvider({
+  signer: await InMemorySigner.fromSecretKey('your_private_key', 'your_passphrase'),
+});
+```
+
+### Using a testnet faucet key
+
+To load a faucet key (available from https://faucet.tzalpha.net/) for working a public testnet use the `importKey` function.
+can do so as follows:
```

Comment:
"can do so as follows:" -> "You can do so as follows:"

Reply:
Fixed, thanks!